documents/snapshots/mip37.md

```diff
@@ -29,4 +29,4 @@ Conclusion
 As stablecoin specialists, Steakhouse Financial has cultivated significant experience and network within DeFi. Leveraging our insights from helping Dai (MakerDAO) and stETH (LidoDAO) grow, we look forward to supporting relatively new assets like USDM and PYUSD gain wide adoption. We are also excited to bring a credible alternative to the existing USDT borrowing/lending market, using our bluechip “dual engine” strategy.
 Steakhouse has begun educating investment funds, wallets and fintech players on MetaMorpho with the successful launch of the steakUSDC vault. We look forward to continuing on that journey with our three new proposed vaults and bringing value to the Morpho ecosystem.
 
-The proposal was voted on and accepted on 25 February 2024.
\ No newline at end of file
+The proposal was voted on and accepted on 25 February 2024, but Quorum wasn't reached.
```

Comment:
Feels weird to say accepted but Quorum not reached no? 
Also, would write it: 
quorum

Reply:
changed phrasing and corrected 'quorum' capital letter!